<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dlgCalBuildTable</class>
 <widget class="QDialog" name="dlgCalBuildTable">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>655</width>
    <height>313</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Calibration - Build Table</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0" colspan="4">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Calibrate the Build Table to be flush with the PDMS Vat Floor</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QSplitter" name="splitter">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <widget class="QLabel" name="label">
         <property name="palette">
          <palette>
           <active>
            <colorrole role="WindowText">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>0</red>
               <green>0</green>
               <blue>127</blue>
              </color>
             </brush>
            </colorrole>
           </active>
           <inactive>
            <colorrole role="WindowText">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>0</red>
               <green>0</green>
               <blue>127</blue>
              </color>
             </brush>
            </colorrole>
           </inactive>
           <disabled>
            <colorrole role="WindowText">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>120</red>
               <green>120</green>
               <blue>120</blue>
              </color>
             </brush>
            </colorrole>
           </disabled>
          </palette>
         </property>
         <property name="toolTip">
          <string>Instructions</string>
         </property>
         <property name="whatsThis">
          <string>Instructions - For more detailed information visit www.b9creator.com</string>
         </property>
         <property name="text">
          <string>The purpose of the Build Table Calibration is to ensure the Build Table is flush against the PDMS floor of the VAT when commanded to the &quot;Zero&quot; height position.

This calibration should be preformed when a new or re-coated VAT is put into service.</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QCheckBox" name="checkBoxStep1">
         <property name="toolTip">
          <string>Step 1</string>
         </property>
         <property name="whatsThis">
          <string>Step 1 of 8</string>
         </property>
         <property name="text">
          <string>Step 1- Ensure the VAT is installed and the Sweeper is REMOVED.</string>
         </property>
        </widget>
        <widget class="QCheckBox" name="checkBoxStep2">
         <property name="toolTip">
          <string>Step 2</string>
         </property>
         <property name="whatsThis">
          <string>Step 2 of 8</string>
         </property>
         <property name="text">
          <string>Step 2 - Loosen all four build table adjustment screws using a 7/64&quot; allen (hex) wrench and install it on the arm.</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButtonHomeStep3">
         <property name="toolTip">
          <string>Step 3</string>
         </property>
         <property name="whatsThis">
          <string>Step 3 of 8</string>
         </property>
         <property name="text">
          <string>Step 3 - Click here to command the printer to locate the Home position (Arm and VAT will move!)</string>
         </property>
         <property name="autoDefault">
          <bool>true</bool>
         </property>
         <property name="default">
          <bool>false</bool>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButtonZeroStep4">
         <property name="toolTip">
          <string>Step 4</string>
         </property>
         <property name="whatsThis">
          <string>Step 4 of 8</string>
         </property>
         <property name="text">
          <string>Step 4 - Click here to command the printer to open the VAT and lower the build table to &quot;Zero&quot;</string>
         </property>
        </widget>
        <widget class="QCheckBox" name="checkBoxStep5">
         <property name="toolTip">
          <string>Step 5</string>
         </property>
         <property name="whatsThis">
          <string>Step 5 of 8</string>
         </property>
         <property name="text">
          <string>Step 5 - Ensure the bottom of the Build Table is sitting flush on the PDMS.  Tighten all four build table adjustment screws.</string>
         </property>
        </widget>
        <widget class="QCheckBox" name="checkBoxStep6">
         <property name="toolTip">
          <string>Step 6</string>
         </property>
         <property name="whatsThis">
          <string>Step 6 of 8</string>
         </property>
         <property name="text">
          <string>Step 6 - REMOVE the Build Table Thumbscrew.  This is important!  Failure to do so may damage the PDMS when the arm lifts.</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButtonRaiseUpStep7">
         <property name="toolTip">
          <string>Step 7</string>
         </property>
         <property name="whatsThis">
          <string>Step 7 of 8</string>
         </property>
         <property name="text">
          <string>Step 7 - Click here to command the printer to raise the arm out of the way.</string>
         </property>
        </widget>
        <widget class="QCheckBox" name="checkBoxStep8">
         <property name="toolTip">
          <string>Step 8</string>
         </property>
         <property name="whatsThis">
          <string>Final Step</string>
         </property>
         <property name="text">
          <string>Step 8 - After the arm is clear, carefully lift the Build Table from the PDMS and fully tighten all four screws.</string>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Status:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="lineEditStatus">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="palette">
      <palette>
       <active>
        <colorrole role="Text">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>0</red>
           <green>0</green>
           <blue>127</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Base">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>204</red>
           <green>204</green>
           <blue>204</blue>
          </color>
         </brush>
        </colorrole>
       </active>
       <inactive>
        <colorrole role="Text">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>0</red>
           <green>0</green>
           <blue>127</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Base">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>204</red>
           <green>204</green>
           <blue>204</blue>
          </color>
         </brush>
        </colorrole>
       </inactive>
       <disabled>
        <colorrole role="Text">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>120</red>
           <green>120</green>
           <blue>120</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Base">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>240</red>
           <green>240</green>
           <blue>240</blue>
          </color>
         </brush>
        </colorrole>
       </disabled>
      </palette>
     </property>
     <property name="toolTip">
      <string>Status</string>
     </property>
     <property name="whatsThis">
      <string>Status - reports if printer is in motion or read for input.</string>
     </property>
     <property name="text">
      <string>Ready...</string>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>251</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="3">
    <widget class="QPushButton" name="pushButtonDone">
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButtonHomeStep3</sender>
   <signal>clicked()</signal>
   <receiver>dlgCalBuildTable</receiver>
   <slot>on_Step3()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>257</x>
     <y>231</y>
    </hint>
    <hint type="destinationlabel">
     <x>319</x>
     <y>239</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonZeroStep4</sender>
   <signal>clicked()</signal>
   <receiver>dlgCalBuildTable</receiver>
   <slot>on_Step4()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>252</x>
     <y>276</y>
    </hint>
    <hint type="destinationlabel">
     <x>319</x>
     <y>239</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonRaiseUpStep7</sender>
   <signal>clicked()</signal>
   <receiver>dlgCalBuildTable</receiver>
   <slot>on_Step7()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>252</x>
     <y>321</y>
    </hint>
    <hint type="destinationlabel">
     <x>319</x>
     <y>239</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonDone</sender>
   <signal>clicked()</signal>
   <receiver>dlgCalBuildTable</receiver>
   <slot>done()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>597</x>
     <y>281</y>
    </hint>
    <hint type="destinationlabel">
     <x>321</x>
     <y>150</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxStep1</sender>
   <signal>clicked(bool)</signal>
   <receiver>dlgCalBuildTable</receiver>
   <slot>on_Step1(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>318</x>
     <y>68</y>
    </hint>
    <hint type="destinationlabel">
     <x>321</x>
     <y>150</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxStep2</sender>
   <signal>clicked(bool)</signal>
   <receiver>dlgCalBuildTable</receiver>
   <slot>on_Step2(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>318</x>
     <y>90</y>
    </hint>
    <hint type="destinationlabel">
     <x>321</x>
     <y>150</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxStep5</sender>
   <signal>clicked(bool)</signal>
   <receiver>dlgCalBuildTable</receiver>
   <slot>on_Step5(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>318</x>
     <y>168</y>
    </hint>
    <hint type="destinationlabel">
     <x>321</x>
     <y>150</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxStep6</sender>
   <signal>clicked(bool)</signal>
   <receiver>dlgCalBuildTable</receiver>
   <slot>on_Step6(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>318</x>
     <y>190</y>
    </hint>
    <hint type="destinationlabel">
     <x>321</x>
     <y>150</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxStep8</sender>
   <signal>clicked(bool)</signal>
   <receiver>dlgCalBuildTable</receiver>
   <slot>on_Step8(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>318</x>
     <y>240</y>
    </hint>
    <hint type="destinationlabel">
     <x>321</x>
     <y>150</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>findHome()</slot>
  <slot>goZero()</slot>
  <slot>raiseUp()</slot>
  <slot>done()</slot>
  <slot>on_Step1(bool)</slot>
  <slot>on_Step2(bool)</slot>
  <slot>on_Step3()</slot>
  <slot>on_Step4()</slot>
  <slot>on_Step5(bool)</slot>
  <slot>on_Step6(bool)</slot>
  <slot>on_Step7()</slot>
  <slot>on_Step8(bool)</slot>
 </slots>
</ui>
